src/test/java/digital/slovensko/autogram/AutogramTests.java

```diff
@@ -35,10 +35,14 @@ void testSignHappyScenario() {
         var document = new FileDocument("pom.xml");
         var responder = mock(Responder.class);
 
-        autogram.pickSigningKeyAndThen(key
-        -> autogram.sign(new SigningJob(document, parameters, responder), key));
+        try (MockedStatic<TokenDriver> mockedStatic = mockStatic(TokenDriver.class)) {
+            mockedStatic.when(TokenDriver::getAvailableDrivers).thenReturn(Arrays.asList(new FakeTokenDriver("fake")));
```

Comment:
Toto smrdi tym, ze nam tu chyba nejake injektovanie objektu co odpoveda na `getAvailableDrivers`. Mozno by sme to mohli posielat do autogramu ako optional parameter v konstruktore, nasledne tam len posles mock a nemusis mockovat vnutornosti. Hm?